projects/vdk-control-cli/src/vdk/internal/control/rest_lib/rest_client_errors.py

```diff
@@ -12,31 +12,38 @@
 
 
 class ApiClientErrorDecorator:
-    def __init__(self, what="Control Service Error"):
-        self.what = what
+    def __init__(
+        self, what="Control Service Error", consequences="Operation cannot complete."
+    ):
+        self.__what = what
+        self.__consequences = consequences
 
     def _get_error(self, exception: ApiException):
-        # noinspection PyBroadException
         try:
             return json.loads(exception.body)
         except:
             error = {}
             if exception.status == 401:
                 error = dict(
-                    what=self.what,
-                    why="The request has not been applied because it lacks valid authentication credentials.",
-                    consequences="Operation cannot complete.",
-                    countermeasures="Try to login again using VDK CLI login command. "
-                    "Make sure you have permission to execute the given operation.",
+                    what=self.__what,
+                    why=f"The request has not been applied because it lacks valid authentication credentials. "
+                    f"Error returned by control service is {exception.body}",
+                    consequences=self.__consequences,
+                    countermeasures="Try to login again using VDK CLI login command."
+                    "Or set correct api token configuration (see vdk config-help)."
+                    "Make sure you have permission to execute the given operation."
+                    "Investigate (google) the error returned by control service."
+                    "And if all fails try to contact the support team.",
                 )
             if exception.status == 404:
                 error = dict(
-                    what=self.what,
+                    what=self.__what,
                     why="The requested resource cannot be found. "
                     "You may have a spelling mistake or the data job has not been created",
-                    consequences="Operation cannot complete.",
-                    countermeasures="Make sure that the data job name and team name  are spelled correctly (it is is case-sensitive) . "
-                    "Make sure you have run vdk create before doing any other operations on a data job.",
+                    consequences=self.__consequences,
+                    countermeasures="Make sure that the data job name and team name are spelled correctly "
+                    "(it is case-sensitive) . "
+                    "Make sure you have run vdk create --cloud before doing this operation on a data job.",
```

Comment:
```suggestion
                    "Make sure you have run `vdk create --cloud` before doing this operation on a data job.",
```

Reply:
Apostrophes around the command makes it pop up when someone is skim reading the error.